pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/PauselessRealtimeIngestionWithDedupIntegrationTest.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.integration.tests;
+
+import org.apache.pinot.spi.config.table.ingestion.IngestionConfig;
+
+
+public class PauselessRealtimeIngestionWithDedupIntegrationTest extends BaseDedupIntegrationTest {
+  @Override
+  protected IngestionConfig getIngestionConfig() {
+    IngestionConfig ingestionConfig = super.getIngestionConfig();
+    assert ingestionConfig != null;
+    assert ingestionConfig.getStreamIngestionConfig() != null;
+    ingestionConfig.getStreamIngestionConfig().setPauselessConsumptionEnabled(true);
```

Comment:
Should we not enable rest of the pauseless + dedup configs here like parallel consumption policy?